charts/greptimedb-etcd/README.md

```diff
@@ -12,7 +12,7 @@ helm repo add greptime https://greptimeteam.github.io/helm-charts/
 helm repo update
 
 # Deploy greptimedb-etcd in default namespace.
-helm install mydb greptime/greptimedb-etcd -n default --devel
+helm install gt greptime/greptimedb-etcd -n default --devel
```

Comment:
you can use the cluster name(`mydb`) that used in `greptimedb` chart

Reply:
use mydb-etcd release name
```
helm install mydb-etcd greptime/greptimedb-etcd -n default --devel
```